frontend/src/learning/components/ItemDisplay/BreakdownView.js

```diff
@@ -208,12 +222,12 @@ export default class BreakdownView extends React.Component {
      * specified in the urls.
      * @param {[String]} urls 
      */
-    renderBreakdownChar(urls) {
+    renderBreakdownChar(urls, qid) {
         return urls.map( url => 
             (
                 <div key={url}
                     className='breakdown-card box-shadow'>
-                    <BreakdownChar url={url} />
+                    <BreakdownChar url={url} qid={qid} />
```

Comment:
The `BreakdownRad` and `BreakdownChar` class was not intended for displaying on a full page view. Use `RadDisplay` and `CharDisplay` instead.